database/sql/V007__extend_trades_with_txhash.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE trades
+    ADD COLUMN tx_hash bytea NOT NULL,
+    ADD COLUMN settlement_index bigint NOT NULL;
```

Comment:
If these are NOT NULL you might have to think about what to set them to for the records that are already in the table.

Also given that these two columns together will act as a pseudo foreign key into the batches table, I was wondering if it would make sense to replace them with a single column batch_id (but not sure how this might affect the simplicity of this PR in itself).

Reply:
I was under the impression from past experience with Postgres that if you make a column required at a later point in time the database won't complain that those pre existing records could remain as they were. Lucky for us though the settlement index for every batch so far is probably zero anyway (although I would have to double check) so we could simpleton perform a write to update all the null values to zero. I would also have to double check this other assumption about pre existing record that is probably not true (especially for a required field that is involved as a foreign key)

Hopefully the settlement index is zero for all previous settlement events.